src/notifications/screens/notifications.screen.js

```diff
@@ -187,6 +187,7 @@ class Notifications extends Component {
     markAsReadByDispatch(notification.id);
     navigation.navigate('Issue', {
       issueURL: notification.subject.url.replace('pulls', 'issues'),
+      isPR: notification.subject.type === 'PullRequest',
```

Comment:
@lex111 as discussed, this PR takes care of navigating to issues from the notifications screen and you'll prefer to keep it as such. I'm okay with that, we'll just need another PR to add the `isPR` flag to other places in the app that navigates to the `Issue` screen

Regarding that bug you're seeing, let's please open a separate bug for that if we're seeing the issue screen failing because of that message